getting-started.md

```diff
@@ -62,3 +62,26 @@ async fn main() {
 ```
 
 For more examples, see the [examples](examples) directory.
+
+## A TiKV cluster
+
+To use the client, you'll need a TiKV instance to communicate with. In production, this should be a cluster of dedicated servers which are accessed via the network. To get started, you can run a TiKV 'cluster' on your local machine.
```

Comment:
```suggestion
To use the client, you'll need a TiKV instance to communicate with. In production, this should be a cluster of dedicated servers which are accessed via the network. To get started, you can run a TiKV cluster on your local machine.
```

Reply:
The inverted commas indicate this is not really a cluster